internal/params/config.go

```diff
@@ -25,89 +25,95 @@ var EpochTBD = big.NewInt(10000000)
 var (
 	// MainnetChainConfig is the chain parameters to run a node on the main network.
 	MainnetChainConfig = &ChainConfig{
-		ChainID:          MainnetChainID,
-		CrossTxEpoch:     big.NewInt(28),
-		CrossLinkEpoch:   big.NewInt(186),
-		StakingEpoch:     big.NewInt(186),
-		PreStakingEpoch:  big.NewInt(185),
-		QuickUnlockEpoch: big.NewInt(191),
-		FiveSecondsEpoch: big.NewInt(230),
-		EIP155Epoch:      big.NewInt(28),
-		S3Epoch:          big.NewInt(28),
-		ReceiptLogEpoch:  big.NewInt(101),
+		ChainID:           MainnetChainID,
+		CrossTxEpoch:      big.NewInt(28),
+		CrossLinkEpoch:    big.NewInt(186),
+		StakingEpoch:      big.NewInt(186),
+		PreStakingEpoch:   big.NewInt(185),
+		QuickUnlockEpoch:  big.NewInt(191),
+		FiveSecondsEpoch:  big.NewInt(230),
+		RedelegationEpoch: big.NewInt(10000000),
+		EIP155Epoch:       big.NewInt(28),
+		S3Epoch:           big.NewInt(28),
+		ReceiptLogEpoch:   big.NewInt(101),
 	}
 
 	// TestnetChainConfig contains the chain parameters to run a node on the harmony test network.
 	TestnetChainConfig = &ChainConfig{
-		ChainID:          TestnetChainID,
-		CrossTxEpoch:     big.NewInt(0),
-		CrossLinkEpoch:   big.NewInt(2),
-		StakingEpoch:     big.NewInt(2),
-		PreStakingEpoch:  big.NewInt(1),
-		QuickUnlockEpoch: big.NewInt(0),
-		FiveSecondsEpoch: big.NewInt(16500),
-		EIP155Epoch:      big.NewInt(0),
-		S3Epoch:          big.NewInt(0),
-		ReceiptLogEpoch:  big.NewInt(0),
+		ChainID:           TestnetChainID,
+		CrossTxEpoch:      big.NewInt(0),
+		CrossLinkEpoch:    big.NewInt(2),
+		StakingEpoch:      big.NewInt(2),
+		PreStakingEpoch:   big.NewInt(1),
+		QuickUnlockEpoch:  big.NewInt(0),
+		FiveSecondsEpoch:  big.NewInt(16500),
+		RedelegationEpoch: big.NewInt(10000000),
```

Comment:
shall we enable it earlier on testnet ?

Reply:
I will enable it once the testnet current issues are fixed.